larrecodnn/ImagePatternAlgs/Modules/RawWaveformDump_module.cc

```diff
@@ -192,8 +201,10 @@ nnet::RawWaveformDump::RawWaveformDump(fhicl::ParameterSet const& p)
     throw cet::exception("RawWaveformDump")
       << "Only one of DigitModuleLabel and WireProducerLabel should be set";
   }
-
-  //this->reconfigure(p);
+  fRandFlat = std::make_unique< CLHEP::RandFlat >
+              (createEngine(art::ServiceHandle<rndm::NuRandomService>{}->declareEngine(
+                            instanceName, p, "SeedForRawWaveformDump"),
+                            "HepJamesRandom", instanceName));
```

Comment:
Same comment as above--change the type of `fRandFlat` to `CLHEP::RandFlat` and initialize it in the initialization list.

Reply:
took care of this too like above.